cl/users/admin.py

```diff
@@ -76,37 +77,55 @@ class UserAdmin(admin.ModelAdmin, AdminTweaksMixin):
         "profile__stub_account",
     )
     search_help_text = (
-        "Search Users by username, first name, last name, or email."
+        "Search Users by username, first name, last name, email, or pk."
     )
     search_fields = (
         "username",
         "first_name",
         "last_name",
         "email",
+        "pk",
     )
 
     def change_view(self, request, object_id, form_url="", extra_context=None):
         """Add links to related event admin pages filtered by user/profile."""
         extra_context = extra_context or {}
         user = self.get_object(request, object_id)
-
-        extra_context["proxy_events_url"] = build_admin_url(
-            UserProxyEvent,
-            {"pgh_obj": object_id},
+        profile_id = (
+            user.profile.pk if user and hasattr(user, "profile") else None
+        )
+        has_tags = (
+            user and hasattr(user, "user_tags") and user.user_tags.count()
```

Comment:
We'll always have a profile so no need to check that. I think for the tags it's fine to always show it because otherwise I'll get confused, "Wasn't there a button here before??" Maybe just keep these simple, and same for search queries?